projects/angular/src/layout/breadcrumbs/breadcrumbs.spec.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Location } from '@angular/common';
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { ClrLayoutModule } from '../layout.module';
+
+@Component({
+  template: ` <clr-breadcrumbs [items]="menuItems"></clr-breadcrumbs> `,
+})
+class TestComponent {
+  menuItems = [
+    { label: 'Home', url: '/home' },
+    { label: 'Parent Page', url: '/parent' },
+    { label: 'Current Page', url: '/child' },
+    { label: 'Grandchild Page', url: '/grandchild' },
+    { label: 'Last Page', url: '/last' },
+  ];
+  isExpanded = false;
+  handleItemClick = breadcrumb => {
+    return breadcrumb;
+  };
+}
+let breadcrumbs: DebugElement;
+let breadcrumbList: DebugElement;
+
+describe('ClrBreadcrumbs', () => {
+  let component: TestComponent;
+  let fixture: ComponentFixture<TestComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [
+        ClrLayoutModule,
+        RouterTestingModule.withRoutes([
+          { path: 'home', component: TestComponent },
+          { path: 'child', component: TestComponent },
+        ]),
+      ],
+      declarations: [TestComponent],
+    }).compileComponents();
```

Comment:
```suggestion
  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [
        ClrLayoutModule,
        RouterTestingModule.withRoutes([
          { path: 'home', component: TestComponent },
          { path: 'child', component: TestComponent },
        ]),
      ],
      declarations: [TestComponent],
    });
```

The `compileComponents` call is not needed.